spec/system/distributions_by_county_system_spec.rb

```diff
@@ -34,6 +34,32 @@
       expect(page).to have_text("25", count: 4)
       expect(page).to have_text("$262.50", count: 4)
     end
+
+    it("works for prior year") do
+      @distribution_current = create(:distribution, :with_items, item: item_1, organization: user.organization, partner: partner_1, issued_at: issued_at_present)
+      @distribution_last_year = create(:distribution, :with_items, item: item_1, organization: user.organization, partner: partner_1, issued_at: issued_at_last_year)
+
+      visit_distribution_by_county_with_specified_date_range("Prior Year")
+
+      partner_1.profile.served_areas.each do |served_area|
+        expect(page).to have_text(served_area.county.name)
+      end
+      expect(page).to have_text("25", count: 4)
+      expect(page).to have_text("$262.50", count: 4)
+    end
+
+    it("works for last 12 months") do
+      @distribution_current = create(:distribution, :with_items, item: item_1, organization: user.organization, partner: partner_1, issued_at: issued_at_present)
+      @distribution_last_year = create(:distribution, :with_items, item: item_1, organization: user.organization, partner: partner_1, issued_at: issued_at_last_year)
+
+      visit_distribution_by_county_with_specified_date_range("Last 12 Months")
+
+      partner_1.profile.served_areas.each do |served_area|
+        expect(page).to have_text(served_area.county.name)
+      end
+      expect(page).to have_text("25", count: 4)
+      expect(page).to have_text("$262.50", count: 4)
+    end
```

Comment:
Aren't these tests identical? Ideally to test this we'd want to have a distribution that _is_ included and one that's _not_ included, specifically for these 2 blocks of time.

Reply:
Yeah.... I was curious about that. I will dig in a bit deeper to see what I can figure out.

Reply:
Sorry! I haven't forgotten about this.... Work has picked up a bit so been eating up a lot of my time at the moment!